build.sh

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+
+#!/bin/sh
+
+# Pull down the index generator/validator if needed
+if [ ! -d "./registry-support" ]; then
+  echo "Cloning index-generator tool"
+  git clone https://github.com/devfile/registry-support.git ./registry-support
+  if [ ! $? -eq 0 ]; then
+    echo "Failed to clone index-generator tool"
+    exit 1
+  fi
+  echo "Successfully pulled the index-generator tool\n"
+fi
+
+# Build the index generator/validator
+echo "Building index-generator tool"
+cd ./registry-support/index/generator
+./build.sh
+if [ ! $? -eq 0 ]; then
+  echo "Failed to build index-generator tool"
+  exit 1
+fi
+echo -e "Successfully built the index-generator tool\n"
+cd "$OLDPWD"
```

Comment:
Seems like we should define the `OLDPWD` before building the index generator?

Reply:
So, it actually is a predefined shell environment variable: 
<img width="682" alt="Screen Shot 2020-11-19 at 9 50 50 AM" src="https://user-images.githubusercontent.com/6880023/99681953-b8214000-2a4c-11eb-930b-87ba117116f1.png">

I was trying to avoid using `cd -`, as it always prints the old directory, and found `$OLDPWD` as an alternative.

Reply:
Oh, Today I Learned!